### PR TITLE
feat:  onboarding enhancement

### PR DESCRIPTION
**Pull request by Lymah123**
This PR serves as submission to issue:
- #251

### CODE REVIEW

**Comment by Lymah123** [comment]
@phyleria , this is the PR to the issue I raised. I will be waiting for your review.

**Review by phyleria — CHANGES_REQUESTED** [review]
Hi @Lymah123 

Can you add this as a section on the READMe file instead?

**Comment by Lymah123** [comment]
> Hi @Lymah123
> 
> Can you add this as a section on the READMe file instead?

Hi @phyleria, okay. I will do that.

**Comment by Lymah123** [comment]
Hi @phyleria , I have made the changes.

**Review by phyleria — CHANGES_REQUESTED** [review]
Hi @Lymah123 

Just a few other changes, customize the description in a way that explains to the contributors how to contribute via Gitpod/Codespaces. An example is this: 

## Contributing with [Gitpod](https://www.gitpod.io/) or [GitHub Codespaces](https://github.com/features/codespaces)

 ```Input Descriptions on how to set up both```

**Comment by Lymah123** [comment]
Hi @phyleria, you can review again. I will be waiting for your feedback. Thanks!

File: README.md [hunk 1]
```diff
@@ -102,7 +102,55 @@ Contributing to Chimoney's Hacktoberfest is easy! Follow these steps:
 
 5. Create a pull request explaining your contribution.
 
-## 🧿 Issues
+## Onboarding enhancement
+
```
Comment (uchibeke) [inline README.md:106]: Should there be a gitpod icon/button which when clicked auto sets up this project on gitpod? That’s what I’ve seen other projects do. We don’t want to teach people how to use gitpod but instead include the Gitpod and spaces but and a few lines of instructions 

maybe link out to official docs of both pod and spaces

**Comment by Lymah123** [comment]
Alright. I will work on that. 

@phyleria , let me know your suggestion as well.

**Comment by phyleria** [comment]
I agree with the suggestion as well. For reference you can also check out how we implemented the same on this [repo's READMe](https://github.com/Chimoney/Community-Website)

**Comment by Lymah123** [comment]
> I agree with the suggestion as well. For reference you can also check out how we implemented the same on this [repo's READMe](https://github.com/Chimoney/Community-Website)

Okay.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,7 +5,18 @@ Before contributing to this repository, please read this [article](https://hashn
...
+- [Gitpod](https://www.gitpod.io/)
+
+   ## Contribute via Gitpod:
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/Chimoney/Community-projects)
+
+
+- Check out how to get started [here](https://www.gitpod.io/docs/introduction)
+
+## Contributing via CodeSpaces:
+[![Open in Visual Studio Codespaces](https://github.com/Lymah123/chimoney-community-projects/commit/995fa27bdc463d3ddcff6bbab64c34060ec77858)](https://codespaces.new/Chimoney/Community-Website)
```
Comment (uchibeke) [inline CONTRIBUTING.md]: Does this opens specific commit? If so, why?

Comment (Lymah123) [reply CONTRIBUTING.md]: > Does this opens specific commit? If so, why?

I am still working on it, please.

Comment (Lymah123) [reply CONTRIBUTING.md]: Hi @uchibeke , you can check again now. Thank you!

**Comment by Lymah123** [comment]
Hi, @uchibeke and @phyleria . Any update on the PR?

**Review by phyleria — CHANGES_REQUESTED** [review]
Hi @Lymah123 

Kindly remove this part, I don't think it's relevant.

**Review by phyleria — CHANGES_REQUESTED** [review]
Also make this changes, on the main READMe file immediately after the `projects setup` section

**Comment by Lymah123** [comment]
Hi @phyleria , do you mean I should delete everything under this PR?

**Comment by phyleria** [comment]
Hi @Lymah123 

No, I meant to remove the `Onboarding enhancement` description and move the rest of the changes to the READMe file. Does it make sense now?

**Comment by Lymah123** [comment]
Hi @phyleria , You can check again now. Is that what you asked me to do?

**Comment by Lymah123** [comment]
Hi @phyleria .

Any update?

**Pull request closed without merging**